.buildkite/bk.integration-fips.pipeline.yml

```diff
@@ -39,23 +45,19 @@ steps:
       - elastic/vault-docker-login#v0.5.2:
           secret_path: 'kv/ci-shared/platform-ingest/elastic_docker_registry'
 
-  - label: Start ESS stack for FIPS integration tests
+  - label: Start ESS stack for FIPS integration tests using oblt-cli
     key: integration-fips-ess
     depends_on:
       - integration-fips-cloud-image
     env:
       ASDF_TERRAFORM_VERSION: 1.9.2
       TF_VAR_integration_server_docker_image: "docker.elastic.co/beats-ci/elastic-agent-cloud-fips:git-${BUILDKITE_COMMIT:0:12}"
```

Comment:
@v1v Will this image be used as the integrations server when using oblt-cli?

Reply:
That's something I need to discuss with our team internally - this PR is still in a very early stage, thanks for the heads up 🙇